itests/shellenv.feature

```diff
@@ -0,0 +1,13 @@
+Feature: env variables
+
+Scenario: JBANG_RUNTIME_SHELL available
+When command('jbang env@jbangdev jbang')
+Then match err contains "^JBANG_RUNTIME_SHELL"
+
+Scenario: JBANG_STDIN_NOTTY available
```

Comment:
@quintesse  what is the use of this variable?

Reply:
This tells JBang that we don't have a console/terminal connected to stdin. Java can't detect this by itself.

Reply:
I believe this is a pretty good approximation, in case you ever need it:

```java
boolean isAnsi = System.console()!=null && 
  System.getProperty("os.name").toLowerCase(Locale.ENGLISH).contains("win");
```

Reply:
@dsyer that would only work on *nix systems and we need it to work for Windows too.

Reply:
But where in code are we using that env var?

Reply:
See `Util.haveConsole()` which is used by `ConsoleInput.get()` which is used by `Util.askInput()` to detect if we can ask the user for input on the console or if we should use a GUI.

Reply:
kk